components/sidebar/unread_channels/index.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getUnreadChannels} from 'selectors/views/channel_sidebar';
+
+import {GlobalState} from 'types/store';
+
+import UnreadChannels from './unread_channels';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+        unreadChannels: getUnreadChannels(state),
```

Comment:
This could theoretically be passed from the `SidebarChannelList`, but I decided to call the selector again for the sake of clarity.